docs/sources/experiments/cohort-labels.md

```diff
@@ -0,0 +1,493 @@
+
+```python
+from triage.experiments import SingleThreadedExperiment
+from triage import create_engine
+import yaml
+
+with open('<your_experiment_config.yaml>') as fd:
+    experiment_config = yaml.load(fd)
+experiment = SingleThreadedExperiment(
+    experiment_config=experiment_config,
+    db_engine=create_engine(...),
+    project_path='./'
+)
+experiment.generate_labels()
+print(experiment.labels_table_name)
+```
+
+This will generate the labels for each as-of-date in your experiment. The table name is autogenerated by the Experiment, and you can retrieve it using the `labels_table_name` attribute of the Experiment. Here, as in option 1, you can look at the data in your SQL browser of choice.
+
+These options should be enough to test your labels in isolation. How the labels shows up in matrices is also dependent on its interaction with the cohort, and later we'll show how to test that.
+
+
+## Combining Cohorts and Labels to make Matrices
+Looking at the cohort and labels tables in isolation doesn't quite get you the whole picture. They are combined with features to make matrices, and some of the functionality (e.g. `include_missing_labels_in_train_as`) isn't applied until the matrices are made for performance/database disk space purposes.
+
+How does this work? Let's look at some example cohort and label tables.
+
+### Cohort
+
+entity_id | as_of_date
+------------ | -------------
+25 | 2016-01-01
+44 | 2016-01-01
+25 | 2016-02-01
+44 | 2016-02-01
+25 | 2016-03-01
+44 | 2016-03-01
+60 | 2016-03-01
+
+### Label
+entity_id | as_of_date | label
+------------ | ------------- | ------------- 
+25 | 2016-01-01 | True
+25 | 2016-02-01 | False
+44 | 2016-02-01 | True
+25 | 2016-03-01 | False
+44 | 2016-03-01 | True
+60 | 2016-03-01 | True
+
+Above we define three total cohorts, on `2016-01-01`, `2016-02-01`, and `2016-03-01`. The first two cohorts have two entities each and the last one has a new third entity. For the first cohort, only one of the entities has an explicitly defined label (meaning the label query didn't return anything for them on that date).
+
+For simplicity's sake, we are going to assume only one matrix is created that includes all of these cohorts. Depending on the experiment's temporal configuration, there may be one, many, or all dates in a matrix, but the details here are outside of the scope of this document.
+
+In general, the index of the matrix is created using a left join in SQL: The cohort table is the left table, and the labels table is the right table, and they are joined on entity id/as of date. So all of the rows that are in the cohort but not the labels table (in this case, just entity 44/date 2016-01-01) will initially have a null label.
+
+The final contents of the matrix, however, depend on the `include_missing_labels_in_train_as` setting.
+
+### Inspections-Style (don't include missing labels)
```

Comment:
This is confusing, because *we are including those rows as* `NULL`... Is it possible to phrase this differently?